qemu/tests/cache_sizes_test.py

```diff
@@ -0,0 +1,30 @@
+import logging
+
+from qemu.tests.qemu_disk_img import QemuImgTest
+
+
+def run(test, params, env):
+    """
+    Cache sizes test for a guest.
+
+    1. Boot a guest up with different cache sizes.
+    2. Check writing data to the guest works fine.
+    3. Shut the guest down.
+
+    :param test: Qemu test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    file = params["guest_file_name"]
+    initial_tag = params["images"]
+    cache_sizes = params["cache_sizes"].split()
+
+    logging.info("Boot a guest up from initial image: %s, and create a"
+                 " file %s on the disk.", initial_tag, file)
+    for cache_size in cache_sizes:
+        params["drv_extra_params"] = "cache-size=%s" % cache_size
+        base_qit = QemuImgTest(test, params, env, initial_tag)
```

Comment:
Please import from another case, in your case `QemuImgTest`

Reply:
Updated, thanks. Review again, please.